astropy/io/fits/diff.py

```diff
@@ -281,7 +298,15 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
         self.ignore_fields = set(k.upper() for k in ignore_fields)
 
         self.numdiffs = numdiffs
-        self.tolerance = tolerance
+        self.reltol = reltol
+        self.abstol = abstol
+
+        if tolerance is not None:
+            warnings.warn("Named argument `tolerance` is deprecated "
+                "and will be removed in future releases. Use `reltol` instead.")
```

Comment:
This should be formatted a bit difference. Grep the source for `AstropyDeprecationWarning` for some other examples.